src/component/panels/RangesPanel/RangesHeader.tsx

```diff
@@ -138,15 +138,10 @@ function RangesHeader({
     if (Array.isArray(ranges?.values) && ranges.values.length > 0) {
       const { originFrequency: observedFrequency, nucleus } = info;
 
-      const nbDecimalDelta = getNumberOfDecimals(
-        rangesPreferences.deltaPPM.format,
-      );
-      const nbDecimalJ = getNumberOfDecimals(rangesPreferences.deltaHz.format);
-
       const result = rangesToACS(ranges.values, {
         nucleus, // '19f'
-        nbDecimalDelta, // 2
-        nbDecimalJ, // 1
+        nbDecimalDelta: rangesPreferences.deltaPPM.format,
```

Comment:
This property should be renamed. Forget about the breaking change in nmr-processing.
- deltaFormat
- jFormat

Reply:
deltaFormat and couplingFormat was the names